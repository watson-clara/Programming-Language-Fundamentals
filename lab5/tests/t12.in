findall(Z, child(Z, 'Lady Diana Spencer'), X),
print(X),
print('\n'),
halt.
