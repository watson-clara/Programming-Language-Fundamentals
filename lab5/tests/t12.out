[Prince William: Duke of Cambridge, Prince Henry of Wales]
